### PR TITLE
quote_of_the_day: Make spice result responsive in design on small screen devices

### PR DESCRIPTION
**Pull request by NaveenKarippai**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`quote_of_the_day: responsive design output on small screen devices`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{SpiceRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

On `quote_of_the_day` spice, the output clipped the quote icon on
small screen devices. Also, the author name was misaligned on small
screen devices.

Provide an overview of the changes this pull request introduces.

* quote icon hidden on small screen devices using CSS styling

* author name aligned correctly on small screen devices

* CSS media queries were used

###### Which issues (if any) does this fix?

See also: #2758

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@moollaza)

---

Instant Answer Page: https://duck.co/ia/view/quote_of_the_day

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @pnpninja

### CODE REVIEW

**Comment by daxtheduck** [comment]
@NaveenKarippai Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
Deploying to beta...

**Comment by moollaza** [comment]
@NaveenKarippai this still looks a little off. The top margin is too big and the left margin seems odd too.

I would suggest you borrow the CSS used in the BrainyQuote Spice -- it gracefully handles the mobile UI and keeps the quote too:

```css
.is-mobile .brainy__quote--container {
    margin-left: 0px;
    margin-bottom: 0.5em;
}

.is-mobile .zci--brainy_quote .zci__more-at {
    margin-left: 40px;

}

.is-mobile .zci--brainy_quote .brainy__attribution {
    margin-left: 40px;
}
```

**Comment by NaveenKarippai** [comment]
@moollaza Thanks for the review! we have a new changeset here with suggested changes.

**Comment by moollaza** [comment]
@NaveenKarippai Thanks! This LGTM :+1:

File: share/spice/quote_of_the_day/quote_of_the_day.css [hunk 1]
```diff
@@ -1,55 +1,55 @@
-.zci-quote_of_the_day--container {
+.quote_of_the_day_container {
```
Comment (moollaza) [inline share/spice/quote_of_the_day/quote_of_the_day.css]: These should really be namespaced with `.zci--quote_of_the_day .quote_of_the_day_container` -- if anything I would simplify the other class names: e.g. `qotd__container`

**Pull request merged**